src/Raven.Server/Smuggler/Documents/Handlers/SmugglerHandler.cs

```diff
@@ -732,6 +735,8 @@ await using (var gzipStream = new GZipStream(section.Body, CompressionMode.Decom
                     }, operationId, token: token).ConfigureAwait(false);
 
                 await WriteImportResultAsync(context, result, ResponseBodyStream());
+                
+                LogTaskToAudit("import-data", operationId, configuration: null);
```

Comment:
can we use Operations.OperationType.Databaseimport instead of "import-data"?

Reply:
Done